static/templates/actions_popover_content.handlebars

```diff
@@ -21,6 +21,15 @@
     </a>
   </li>
 
+  {{#if message.edit_history}}
+  <li>
+    <a href="#" class="view_edit_history" data-msgid="{{message.id}}">
+      <i class="fa-history"></i>
+        {{t "View message history" }}
```

Comment:
I have already added history icon from fontawesome. But, since we are using older version of fontawesome `fa-history` is not working. Shall we update font awesome, thoughts?